packages/swirl-components/src/components/swirl-toggle-button/swirl-toggle-button.css

```diff
@@ -0,0 +1,54 @@
+:host {
+  display: inline-block;
+
+  & * {
+    box-sizing: border-box;
+  }
+}
+
+.button {
+  display: inline-flex;
+  width: 100%;
+  padding: var(--s-space-8) var(--s-space-16);
+  justify-content: center;
+  align-items: center;
+  border: none;
+  border-radius: var(--s-border-radius-base);
+  color: var(--s-text-default);
+  background-color: var(--swirl-ghost-button-background-default);
+  font: inherit;
+  font-weight: var(--s-font-weight-medium);
+  line-height: var(--s-line-height-lg);
+  text-decoration: none;
+  cursor: pointer;
+  gap: var(--s-space-8);
+
```

Comment:
Suggestion: Could be a good idea to prevent it from wrapping and add ellipsis truncation.